src/matchspec.rs

```diff
@@ -430,6 +433,49 @@ impl<S: AsRef<str> + PartialOrd + PartialEq<str> + Into<String>> MatchSpec<S> {
     }
 }
 
+impl MatchSpec<String> {
+    fn is_match(&self, pc: &PackageCandidate) -> bool {
+        self.is_package_version_match(
+            &pc.name,
+            &pc.version.as_ref().unwrap())
```

Comment:
I'm a little concerned about the `unwrap()` here. Can we make it so that this is won't panic if theres a missing version?